application/controllers/admin/export.php

```diff
@@ -27,6 +27,10 @@ function __construct($controller, $id)
         parent::__construct($controller, $id);
 
         Yii::app()->loadHelper('export');
+
+        Yii::import('application.controllers.admin.*');
+        require_once('printablesurvey.php');
```

Comment:
this works better, yes - thanks. 
I was using an example from here: 
http://www.yiiframework.com/doc/guide/1.1/en/extension.integration